### ISSUE TITLE
Translate rholangtut-0.3.md to Spanish

### ISSUE THREAD

**Issue by ICA3DaR5** [rchain/bounties#651]
# Issue

## Original (English) Content
- [rholangtut-0.3.md](https://github.com/rchain/rchain/blob/master/docs/rholang/rholangtut-0.3.md)

## Translated (Spanish) Content
- [rholangtut-0.3_ES.md](https://github.com/ICA3DaR5/rchain/blob/9a6232e4c34c6a4ff0972fb83fc301e0b918d087/docs/rholang/rholangtut-0.3_ES.md)

## Tasks
- [x] Download the original (English) content.
- [x] Translate the content.
- [x] Check the spelling.
- [x] Review each other's work.
- [x] Have them reviewed by others with knowledge of English and Spanish.
- [x] Create a [Pull Request](https://github.com/rchain/rchain/pull/766).

## Collaborators
- @ICA3DaR5: 40%
- @zero-andreou: 40%
- @michaelizer : 20%

---

### For all issues
[Keep italic labels, replace brackets with answer]

*Describe how the building and implementation of this task will benefit the RChain Membership or the RChain blockchain:* 

More people reading the blog posts in other languages means good marketing.

*Estimated Budget of Task:* **$1200**  \
*Estimated Timeline Required to Complete the Task:* **13 days**  \
*Measure of Completion:* **100%**

--  \
***Legal***  \
*Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.*

### For Translators

  - Why this document?
    - It's important to translate because ____
    - [ ] It's an [RChain blog](https://medium.com/rchain-cooperative) item
    - [x] It's on https://developer.rchain.coop/ that has been endorsed
          for translation by @Jake-Gillberg, @MParlikar, or @KellyAtPyrofex 
  - How can people who do not speak this language be assured of the quality of the translation? (all are required)
    - [x] several reviewers are involved (_assign the ticket to them_)
    - [x] at least two of them have high proficiency in (the target language)
    - [x] at least two of them have high proficiency in English
    - [x] at least two of them have either
       - [x] significant reputation (e.g. a diploma) at stake; source: Diploma
       - [x] a track record of quality work on rchain bounties: #244, #265, #286, #287, #288, #292, #299, #345, #363, #390, #447, #475, #540, #472, #497, #585, #597, #621, #586, #688, #650, #652 
  - What about maintenance? (choose one)
    - [x] the source document is stable; further translation work is not expected
    - [ ] our plan to keep the translation up to date is: ____

**Comment by AyAyRon-P** [comment]
@ICA3DaR5 @zero-andreou @michaelizer1 

Hello, can you we close this out or are we still waiting on a PR merge to happen? 

Please provide a status update to this issue so we can proceed accordingly. 

cc  @pavlos1851

**Comment by ICA3DaR5** [comment]
@AyAyRon-P 

The translation on the [Pull Request](https://github.com/rchain/rchain/pull/766) has been done by @michaelizer1 and @kjmcqueen 

I've asked for a merge. I had previously created another [Pull Request](https://github.com/rchain/rchain/pull/750) and was told by @KellyatPyrofex to create another Pull Request, so I did!

[Comment](https://github.com/rchain/rchain/pull/750#issuecomment-391058040)

**Comment by InternautNeven** [comment]
A PR je pull request, hm možda da maknemo pull request, to ću preksutra pa
napiši da ćemo PR maknuti i poslat novu verziju uskoro

**Comment by ICA3DaR5** [comment]
@InternautNeven I tried to do a Google translate on your comment, but I cannot understand what you mean exactly :)

**Comment by InternautNeven** [comment]
Sorry, i send it by mistake

**Comment by dckc** [comment]
rewarded in 201805
